lib/Doctrine/ODM/PHPCR/Mapping/ClassMetadata.php

```diff
@@ -329,6 +329,13 @@ class ClassMetadata implements ClassMetadataInterface
     public $parentClasses = array();
 
     /**
+     * READ-ONLY: Child class restrictions.
+     *
+     * @var array
+     */
+    public $childRestrictions = null;
```

Comment:
child or children? not sure which is the correct way. its the restriction on each child...